addons/mail/static/src/web/chatter.xml

```diff
@@ -86,12 +84,12 @@
                             this document
                         </t>
                     </b>
-                </div>
+                </small>
             </t>
             <t t-if="state.thread.composer.type === 'message'" t-set="placeholder">Send a message to followers...</t>
             <t t-else="" t-set="placeholder">Log an internal note...</t>
             <t t-set="type" t-value="state.thread.composer.type === 'message' ? 'message' : 'note'"/>
-            <SuggestedRecipientsList t-if="props.hasFollowers and state.thread.composer.type !== 'note'" className="'px-3 pb-1 pt-2'" styleString="'margin-left:48px;'" thread="state.thread"/>
+            <SuggestedRecipientsList t-if="props.hasFollowers and state.thread.composer.type !== 'note'" className="'px-3'" styleString="'margin-left:48px;'" thread="state.thread"/>
```

Comment:
I wonder why those extra paddings were added at all, since they weren't there before the refactoring.
Note that there is still an important padding around the composer for some mysterious reason.